app.js

```diff
@@ -43,14 +50,14 @@ const config = {
   // VSS Settings: See https://github.com/pathikrit/node-visionect
   // Note: This whole section can be removed and things will still work e.g. if you are using the Joan portal
   visionect: {
-    apiServer: 'https://pathikrit-1.dk.visionect.com:8081',
-    apiKey: process.env.visionectApiKey,
-    apiSecret: process.env.visionectApiSecret
+    apiServer: process.env.VISIONECT_API_SERVER || 'https://pathikrit-1.dk.visionect.com:8081',
+    apiKey: process.env.VISIONECT_API_KEY,
+    apiSecret: process.env.VISIONECT_API_SECRET
   }
 }
 
 /** Returns last n days (including today), if timezone is not specified we assume the earliest timezone i.e. UTC+14 */
-const recentDays = (n, timezone = 'Etc/GMT-14')  => _.range(n).map(i => dayjs().tz(timezone).subtract(i, 'days').format('YYYY-MM-DD'))
```

Comment:
I don't think `config.timezone` is required here. Its not a config but a constant per-se. It _has_ to be the "earliest" timezone so we can premeptively fetch the "first" newspaper for "tomorrow" for any timezone on Earth.

Reply:
Updated.